graphs/load_trafalgar_square.py

```diff
@@ -1,12 +1,13 @@
 """Short script used for loading a target graph using osmnx, updating the costs of the edges using up-to-date pollution data, and storing that graph in .gt format"""
+# TODO now that we have the update_graph entrypoint can we remove this?
```

Comment:
Remove file?

Reply:
We could do the plotting and savefig for the overlay? Once that is done then yes there's no need for load_trafalgar.